configs/templates/skylab-dev/spack.yaml

```diff
@@ -24,7 +24,7 @@ spack:
 
     # Individual packages
     - bacio@2.4.1
-    - bison@3.8.2
+    #- bison@3.8.2
```

Comment:
Note to self. Add a comment here that specifying the bison version leads to duplicate packages being built (some using the default/OS bison, some the one requested here). Do after CI tests ran.